README.md

```diff
@@ -532,6 +532,157 @@ ok  	github.com/romshark/jscan/v2	263.004s
 
 </details>
 
+### Intel Xeon E5-2667 v2 - Linux
+
+<details>
```

Comment:
I'd like to differentiate between native and foreign benchmarks because I can't control the circumstances under which those benchmarks were performed. I'm solely relying on trust of correctness here.